tools/release/release.mts

```diff
@@ -59,9 +59,13 @@ if (workspaceVersion === null) {
   process.exit(0);
 }
 
-const publishStatus = await releasePublish({
+const publishProjectsResult = await releasePublish({
```

Comment:
`releasePublish` now always returns a mapping of project names to data (which right now only contains exit codes), rather than one simplistic exit code for the whole set of projects